src/Conversion/ONNXToTOSA/DialectBuilder.cpp

```diff
@@ -100,6 +100,19 @@ Value TosaBuilder::transpose(mlir::Value &value, llvm::ArrayRef<int32_t> perm) {
   return newValue;
 }
 
+Value TosaBuilder::slice(Value &inputConst, llvm::ArrayRef<int64_t> size,
```

Comment:
thanks for hiding the handling of the type inside the builder, that is exactly what builders are for.